tests/output/test_bazefetcher.py

```diff
@@ -1,69 +1,225 @@
 from datetime import datetime
 from pytz import utc
+from pytz import timezone
 import camille.output
 import camille.source
 import numpy as np
 import os
 import pandas as pd
 
+test = 'test'
 
-def assert_correctly_loaded(expected, tag, basedir, t0, t1):
-    bazein = camille.source.bazefetcher(basedir)
+def _assert_correctly_loaded(expected, basedir, t0, t1, tag = test):
+    """
+    Asserts that source.bazefetcher reads exactly what is stored by
+    output.bazefetcher.
+    """
+    bazein = camille.source.bazefetcher(str(basedir))
     result = bazein(tag, t0, t1)
+
     pd.testing.assert_series_equal(expected, result)
 
 
-def test_one_day_one_file(tmpdir):
-    t0 = datetime(2018, 1, 1, 10, tzinfo=utc)
-    t1 = datetime(2018, 1, 1, 13, tzinfo=utc)
-    rng = pd.date_range(t0, t1, freq='H', name='time', closed='left')
-    ts = pd.Series(np.random.randn(len(rng)), name='value', index=rng)
+def _assert_correct_data(basedir, start_date, end_date, expected_index = None,
```

Comment:
It is a Python convention to use `_` prefixes for private functions, but as this is a test script and not a library, I don't think it is necessary to use them. It just makes the lines longer.